user/tlu/module/src/TluRawEvent2StdEventConverter.cc

```diff
@@ -113,20 +113,19 @@ bool TluRawEvent2StdEventConverter::Converting(eudaq::EventSPC d1, eudaq::Standa
 
   auto coarse_ts = static_cast<uint64_t>(d1->GetTimestampBegin() / 25);
 
-  uint8_t coarse_3lsb = 0x3 & coarse_ts;
```

Comment:
Okay, nice, that's a pretty obvious bug... `3bit != 0x3` :rofl:

Reply:
jupp